front-end/src/common/constants.ts

```diff
@@ -155,3 +155,5 @@ export const PING_SRC = {
   DANGER: '/images/game/pings/danger.png',
 };
 
+export const OAUTH_URL = `https://discord.com/api/oauth2/authorize?response_type=code&client_id=737851599778742405&scope=identify&redirect_uri=${encodeURIComponent(process.env.REACT_APP_BACKEND_URL as string)}/api/auth/callback&prompt=identify%20guilds%20guilds.members.read`;
```

Comment:
Does client_id need to be a secret?

Reply:
Discord client id is public. It's in the url for adding a bot the server as well. So it's not hidden